src/index.js

```diff
@@ -0,0 +1,163 @@
+
+const state = {
+    temperature: 68,
+    cityName: "Atlanta",
+    lat: 33.7489924,
+    lon: -84.3902644,
+};
+
+const tempValue = document.getElementById("tempValue");
+
+// Wave 4: Calling APIs
+const convertKtoF = (temp) => {
+  return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const findLatAndLong = () => {
+  //let lat, long;
+  axios
+    .get('http://localhost:5000/location', {
+      params: {
+        q: state.cityName,
+      },
+    })
+    .then((response) => {
+      // console.log(response.data);
+      state.lat = response.data[0].lat;
+      state.lon = response.data[0].lon;
+      getWeather();
+    })
+    .catch((error) => {
+      console.log('Error finding the latitude and longitude:', error.response);
+    });
+};
+
+const getWeather = () => {
+  axios
+    .get('http://localhost:5000/weather', {
+      params: {
+        lat: state.lat,
+        lon: state.lon,
+      },
+    })
+    .then((response) => {
+      const weather = response.data;
+      // console.log(weather);
+      state.temperature = Math.round(convertKtoF(weather.main.temp));
+      // console.log(state.temperature);
+      tempValue.textContent = state.temperature;
+      changeColors();
+      changeLandscape();
+    })
+    .catch((error) => {
+      console.log('Error getting the weather:', error);
+    });
+};
+
+// Wave 2: Increase and Decrease Temperature
+const increaseTempValue = () => {
+    state.temperature += 1;
+    tempValue.textContent = state.temperature;
```

Comment:
You could create this variable assignment in `changeColors` to avoid repeating yourself